modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleBuilder/RulesSupport.es.js

```diff
@@ -98,6 +87,22 @@ const formatRules = (pages, rules) => {
 				true
 			);
 
+			const firstOperandFieldType = getFieldType(
+				condition.operands[0].value,
+				pages
+			);
+
+			if (
+				firstOperandFieldExists &&
+				fieldWithOptions(firstOperandFieldType) &&
+				condition.operands[1].type != 'field'
+			) {
+				const fieldName = condition.operands[0].value;
+				const options = getFieldOptions(fieldName, pages);
```

Comment:
@alinedoleron, please add a break line after this line. Ideally, we should separate variable declarations from statements. 😉